src/bin/pg_autoctl/cli_service.c

```diff
@@ -330,6 +332,26 @@ cli_getopt_pgdata_and_mode(int argc, char **argv)
 				break;
 			}
 
+			case '9':
+			{
+				/* change the signal to send from SIGTERM to SIGKILL */
+				if (!env_exists(PG_AUTOCTL_DEBUG))
+				{
+					log_fatal("Option --sigkill is only available in debug "
+							  "environments");
+					exit(EXIT_CODE_BAD_ARGS);
+					break;
+				}
+
+				if (stop_signal != SIGTERM)
+				{
+					log_fatal("Please use either --fast or --immediate, not both");
```

Comment:
```suggestion
					log_fatal("Please use either --sigkill, --fast or --immediate, not multiple of them");
```

Reply:
I would rather keep `--sigkill` undocumented as I can't imagine a use case in production around it.

Reply:
That makes sense, but this is the error message when already using `--sigkill`. So teh user already knows about it.

Reply:
Ah, yes, indeed, good point. Fixed now.